ddp-workspace/projects/ddp-esc/src/app/components/data-release/data-release.component.ts

```diff
@@ -0,0 +1,79 @@
+import { Component, Inject, OnInit } from '@angular/core';
+import { ToolkitConfigurationService } from 'toolkit';
+
+@Component({
+    selector: 'data-release',
+    template: `
+    <toolkit-header [showButtons]="true"></toolkit-header>
+    <div class="Wrapper">
+        <div class="PageHeader">
+            <div class="PageHeader-background">
+                <div class="PageLayout">
+                    <div class="col-lg-12 col-md-12 col-sm-12 col-xs-12">
+                        <h1 class="PageHeader-title" translate>
+                            DataRelease.Title
+                        </h1>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <article class="PageContent">
+            <div class="PageLayout row">
+                <div class="col-lg-12 col-md-12 col-sm-12 col-xs-12">
+                    <section class="PageContent-section">
+                        <h2 class="PageContent-title" translate>DataRelease.DataBrowser</h2>
+                        <p class="PageContent-text" translate>DataRelease.BrowserText</p>
+                        <p class="PageContent-text" translate>DataRelease.PatientReported</p>
+
+                        <ul class="PageContent-ul">
+                        <li>
+                            <a href="ESC_InitialIntakeSurvey.pdf" target="_blank" class="Link" translate>DataRelease.InitialSurvey</a>
+                        </li>
+                        </ul>
+
+                        <h3 class="PageContent-subtitle" translate>DataRelease.DataUse.Text</h3>
+                        <ul class="PageContent-ul">
+                            <li translate>DataRelease.DataUse.List.Item1</li>
+                            <li translate>DataRelease.DataUse.List.Item2</li>
+                            <li translate>DataRelease.DataUse.List.Item3</li>
+                            <li translate>DataRelease.DataUse.List.Item4</li>
+                        </ul>
+
+                        <p class="PageContent-text">
+                            <span translate>DataRelease.Updated</span>
+                            <a class="Link" [href]="dataEmailHref">{{ dataEmail }}</a>.
+                        </p>
+
+                        <p class="PageContent-text" [innerHtml]="'DataRelease.ThankYou' | translate"></p>
+
+                        <div class="youtube-video">
+                            <iframe [title]="'DataRelease.Video' | translate" class="youtube-video__iframe"
+                                src="https://www.youtube.com/embed/APjftoToMQQ" frameborder="0"
+                                allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+                                allowfullscreen></iframe>
```

Comment:
@zyme, I noticed that Charlotte added the DataRelease page in the ESC project, and I brought these changes into pepper ESC.
But I don't want to commit to our repo 50MB video file. https://github.com/broadinstitute/ddp-gec-angular/commit/00c19df568f65e36c546de0fa9ac2f94f43da375
Can we ask someone to upload video from Charlotte's commit on youtube? As a temporary solution, I added a video from CMI youtube channel.